README.md

```diff
@@ -1,2 +1,80 @@
-# rfcs
-RFCs for changes to Google Cartographer projects
+# Cartographer RFCs
+[Cartographer RFCs]: #cartographer-rfcs
+
+For changes more substantial than simple bugfixes and documentation improvements
+we ask that these be put through a very lightweight design review process.
+
+"RFC" stands for "request for comments" and are used, e.g.
```

Comment:
Is this missing a preposition: `e.g. by [rust-lang]`...?

Reply:
Done